### PR TITLE
Submission of final report and source code of Project 1

### PR DESCRIPTION
**Pull request by TasneemRashid**
Please check the "Final Report" folder for the report and "Source Code" folder for source code of project 1.

### CODE REVIEW

File: projects/project1/Final Report/Students Infromation [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Group member: 1
```
Comment (GezimSejdiu) [inline projects/project1/Final Report/Students Infromation]: There is no need to share the student information on the repo. Please, consider removing the matriculation number at least!

**Review by GezimSejdiu — COMMENTED** [review]
@TasneemRashid not sure what is wrong with this PR, but I'm not able to review the code. I can see only the report and the student information file. The same stand for your repo as well : https://github.com/TasneemRashid/MA-INF-4223-DBDA-Lab/tree/master/projects/project1/Source%20Code .
Please, could you submit the code again that I can review before the final presentation?
Best,

**Comment by TasneemRashid** [comment]
I have created another pull request for the codes. I don't know what was
wrong in the first attempt. I have cloned again my repo and pushed the
files there. Sorry for the inconvenience.

**Comment by GezimSejdiu** [comment]
LGTM

**Pull request merged**